test/acceptance/test/install_tests.go

```diff
@@ -191,4 +192,24 @@ var _ = Describe("Weave GitOps Install Tests", func() {
 			Eventually(errOutput).Should(ContainSubstring(`Error from server (NotFound): namespaces "` + WEGO_DEFAULT_NAMESPACE + `" not found`))
 		})
 	})
+
+	It("Verify wego app is deployed", func() {
+		namespace := "wego-system"
+
+		By("And I have a brand new cluster", func() {
+			_, err := ResetOrCreateCluster(namespace, true)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		installAndVerifyWego(namespace)
```

Comment:
This will need a URL with the new directory structure changes

Reply:
what do you mean?